lookout/style/typos/analyzer.py

```diff
@@ -22,7 +22,8 @@ class IdTyposAnalyzer(Analyzer):
     """
 
     log = logging.getLogger("IdTyposAnalyzer")
-    model_type = None
+    model_type = DummyAnalyzerModel
```

Comment:
This cannot be a Dummy model in the future, because we need to record the existing identifiers somewhere. For now I guess is fine.

Reply:
cc @irinakhismatullina

Reply:
Agree, but right now `train` function returns `DummyAnalyzerModel`, until `train` will be updated we can stick to `DummyAnalyzerModel`

Reply:
I agree, imo we should consider complicating analyzer architecture after we make the one we have work reasonably fine.